core/src/main/scala/org/broadinstitute/dsde/rawls/entities/base/ExpressionValidator.scala

```diff
@@ -20,68 +20,60 @@ trait ExpressionValidator {
   /** validate a MC, skipping optional empty inputs, and return a ValidatedMethodConfiguration */
   def validateMCExpressions(methodConfiguration: MethodConfiguration, gatherInputsResult: GatherInputsResult)(implicit
     executionContext: ExecutionContext
-  ): Future[ValidatedMethodConfiguration] =
```

Comment:
This was returning a future for no obvious reason. 
Since this is primarily being called from a `for` comprehension, I suspect it was to make match up with the other `<-` assignments around it.